ui.config/src/main/content/jcr_root/apps/groovyconsole-config/osgiconfig/config/org.apache.sling.jcr.repoinit.RepositoryInitializer-groovyconsole.config

```diff
@@ -1,9 +1,7 @@
 scripts=[
 "
 create path /conf/groovyconsole/replication(sling:Folder)
-create path /conf/groovyconsole/scripts(sling:Folder)
-
-create path /var/groovyconsole/audit(nt:unstructured)
+create path /var/groovyconsole/audit(sling:Folder)
```

Comment:
is there a reason for deleting the /conf/groovyconsole/scripts? I see there is no .content.xml in ui.content for that specific folder

Reply:
yes, it is created as `nt:folder` through the package (https://jackrabbit.apache.org/filevault/filter.html#Uncovered_ancestor_nodes) if it does not exist yet.  You can change that to `sling:Folder` if you prefer, but I don't think it is necessary. My point is: As it is created through the package always before the repoinit kicks in (at least for AEM 6.5) there is no point in creating the node with repoinit as well.